.rubocop.yml

```diff
@@ -9,9 +9,10 @@ AllCops:
     - '*.yml'
     - 'Gemfile'
     - 'node_modules/**/*'
+    - spec/models/user_spec.rb
```

Comment:
- [ ] Please don't exclude necessary files in your rubocop configuration.

Reply:
We have included it now